release-team/role-handbooks/test-infra/README.md

```diff
@@ -0,0 +1,54 @@
+
+You can also work with @kubernetes/test-infra-maintainers or [test infra oncall](go.k8s.io/oncall) if you are blocked by anything.
+
+### Create CI/Presubmit jobs for the new release
+
+This step should happen in week 6-7, when we create the new release branch.
+
+Most of the release blocking jobs are named with -beta|-stableX, which are mapped to our [release channels](https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs).
+
+TODO(krzyzacy): this section reflects the status of the world today, we are actively looking for simplify the process.
+
+1. Bump build job branches for the [k8s build jobs](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-release/kubernetes-builds.yaml)
+
+1. Create kubekins images for the new release, add a new release target in the [kubekins Makefile](https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e/Makefile)
+
+1. Update release version in the [image bump script](https://github.com/kubernetes/test-infra/blob/master/experiment/bump_e2e_image.sh) and push new kubekins images by running the script. (Note that the runner need to have access to [k8s-testimages](https://pantheon.corp.google.com/home/dashboard?project=k8s-testimages) gcp project)
+
+1. Similarly, make a new Dockerfile for [kubekins-test](https://github.com/kubernetes/test-infra/tree/master/images/kubekins-test) image, this is the image we used for our integration and verify jobs. Also bump the image tags in the [kuberketes_verify scenario](https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_verify.py)
+
+1. grep for `manual-release-bump-required` under [test-infra](https://github.com/kubernetes/test-infra), those are the jobs that need to be manually bumped per release cycle, remap them to the up-to-date branches. Similar to 2, Fork a new version of kubernetes/kubernetes presubmit job, and remove references to the older branches.
+
+1. Okay, now let's update the testgrid config. It's a manual work now, basically you want to find dashboard tabs for release-1.x, and bump that, and the jobs inside, to release-1.(x+1)
+
+1. Finally, update the [release target section](https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs)
+
+Not all the steps need to happen together, some new jobs, like bazel-build/integration/verify will require images to be pushed before they can work properly. 
+
+
+### Ensure the stability of test infra
+
+During the release cycle, especially inside the code freeze, the test infra lead need to actively watch for
+
+1. If the presubmit/CI is failing due to test infra issues (do some initial triage with CI Signal Lead)
+
+1. If PRs are merging into master/release branches
+
+1. Refrain from unrelated changes made to release branch jobs, and breaking changes in testing infrastructure in general
```

Comment:
The way this sentence is structured doesn't match the first 2 bullet points. This isn't something that the test infra lead can actively watch for, it is something they should do.

Reply:
I reworded a bit and moved it out of the bullet points